guides/source/active_storage_overview.md

```diff
@@ -35,6 +35,8 @@ files.
 
 ## Setup
 
+If you are upgrading to Rails 5.2, make sure `require "active_storage/engine"` is declared in `config/application.rb`. This should already be set for new Rails 5.2 applications.
```

Comment:
I can't remember, does `bin/rails app:update` take care of this for users?

Reply:
Good point @kaspth , I forgot about that.  It does, but you still need to uncomment that line. Maybe that's worth mentioning as well.

Reply:
`bin/rails app:update` proposes `require 'rails/all'` if `include_all_railties?` and otherwise a list of frameworks, including `active_storage/engine`; https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/config/application.rb.tt#L11

Reply:
Also, no need to uncomment unless `skip_active_storage` is given.

Reply:
I see that now, not sure how I got the result I did